src/components/common/Page/useFields.tsx

```diff
@@ -64,7 +64,7 @@ export const useFields = (
         .map(({ isVisible, resourceFieldId }) => ({
           ...supportedIds[resourceFieldId],
           // keep the invariant that identity resourceFields are always visible
-          isVisible: isVisible || supportedIds[resourceFieldId].isIdentity,
+          isVisible: isVisible === true || supportedIds[resourceFieldId].isIdentity,
```

Comment:
why we need the explicit equality with `true`?

Reply:
Either that or we can use `Boolean(isVisible)`. This is because `isVisible` could be `undefined`.